compo.py

```diff
@@ -186,20 +186,16 @@ def get_entry_file(uuid: str, filename: str) -> tuple:
     return None, None
 
 
-def get_ranked_entrant_list(which_week: bool) -> str:
-    """Bloc STAR Voting wooooo"""
-
+def verify_votes(which_week: bool) -> None:
```

Comment:
Don't pass `which_week` as a parameter; pass the week itself, since every time you're using `verify_votes` you're already doing `get_week`.